source.txt

```diff
@@ -281,11 +281,11 @@ Table of Contents
        * 414 if the request URI is too long,
        * 416 if Range requests are supported by the server and the Range
              request can not be satisfied,
-       * 401 for all requests that don't have a bearer token with
-             sufficient permissions,
-       * 403 for all requests that either have insufficient scope, e.g. 
-             accessing a <module> for which no scope was obtained, or accessing
-             data outside the user's <storage_root>,
+       * 401 for all requests that require a valid bearer token and
```

Comment:
It was [BEARER], not [OAUTH], sorry (it's also already mentioned at the start of the section).